lib/src/main/java/com/cloudbees/groovy/cps/impl/CpsCallableInvocation.java

```diff
@@ -57,10 +58,21 @@ public Next invoke(Env caller, SourceLocation loc, Continuation k) {
      * @see <a href="https://issues.jenkins-ci.org/browse/JENKINS-31314">JENKINS-31314</a>
      */
     void checkMismatch(Object expectedReceiver, List<String> expectedMethodNames) {
+        String expectedMethodName = expectedMethodNames.get(0);
+        // metaclass invokeMethod
```

Comment:
nit: This comment doesn't really add any more info or context than the code itself, so I'd delete it.